src/plugins/maps_dashboards/public/components/layer_control_panel/layer_control_panel.scss

```diff
@@ -5,3 +5,14 @@
     padding-left: 0.5rem;
   }
 }
+
+.layer-control-title {
+  margin-bottom: 5px;
+  margin-top: 5px;
+}
```

Comment:
Should we use px or % for the sizes? Using px will result in difference in renderings as resolution of device varies.

Reply:
Yes, updated.